Sources/Subs.php

```diff
@@ -829,6 +829,9 @@ function timeformat($log_time, $show_today = true, $offset_type = false, $proces
 	if ($context['server']['is_windows'] && strpos($str, '%e') !== false)
 		$str = str_replace('%e', ltrim(strftime('%d', $time), '0'), $str);
 
+	// fix unkown %l with %I
```

Comment:
I wonder if that is due to a difference in locale. Here is my text code and results:
```
<?php
date_default_timezone_set('America/Edmonton');

$currentLocal = setlocale(LC_ALL, 0); // Get name of current locale

echo $currentLocal, PHP_EOL;
$test = strftime('%l:%M am');
var_dump($test);
```
Result:
```
C
string(8) "12:50 am"
```
What locale does your machine report?